tests/commonUtils.sh

```diff
@@ -93,6 +91,14 @@ function generate_istio_yaml() {
 
     mkdir -p ${dest_dir}
     apply_patch_in_dir "${diff_dir}" "${src_dir}" "${dest_dir}" yaml
+
+    local mhub="$(echo ${MANAGER_HUB_TAG}|cut -f1 -d,)"
+    local mtag="$(echo ${MANAGER_HUB_TAG}|cut -f2 -d,)"
+    local xhub="$(echo ${MIXER_HUB_TAG}|cut -f1 -d,)"
+    local xtag="$(echo ${MIXER_HUB_TAG}|cut -f2 -d,)"
+    sed -i "s#image: .*#image: ${mhub}/proxy:${mtag}#" "${dest_dir}/istio-ingress-controller.yaml"
```

Comment:
What do you mean by use the "right" versions? This is trying to allow users to use "any" version they want.